client/my-sites/domains/style.scss

```diff
@@ -10,3 +11,45 @@
 		align-items: center;
 	}
 }
+
+body.is-section-domains.is-experiment-user {
```

Comment:
The `is-experiment-user` class also feels a bit broad. What does it actually mean? What if there are multiple experiments? Which one does this apply to?

Reply:
Yeah, agree, it's too generic. [Improved it here](https://github.com/Automattic/wp-calypso/pull/73017/commits/ccaaa9cb70837c2e4fd33ab7f6b5656062617386).